common/src/main/java/com/microsoft/identity/common/nativeauth/internal/controllers/NativeAuthMsalController.kt

```diff
@@ -737,8 +730,8 @@ class NativeAuthMsalController : BaseNativeAuthController() {
                     "Access token is expired. Removing from cache..."
                 )
 
-                renewAT(
-                    silentTokenCommandParameters,
+                renewAccessToken(
```

Comment:
We're not removing anything from the cache. That operation seems to be missing from your changes.

Reply:
i have added it